packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html

```diff
@@ -39,5 +33,18 @@ <h2 class="mb-0 d-inline-block text-white" style="font-weight: 400;" data-ng-bin
             <oui-clipboard model="$ctrl.project.project_id"></oui-clipboard>
         </div>
     </div>
+    <div class="d-md-none d-flex py-3">
+        <div class="oui-navbar-menu__back h-100">
+            <button type="button"
+                    class="oui-button oui-button_link oui-button_icon-left"
+                    data-ng-click="$ctrl.toggleProjectsList()">
+                <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
```

Comment:
Having a `span.sr-only` which can give the information that the project list will be toggle could be nice. Isn't it?